cargo-lock/src/bin/cargo-lock/main.rs

```diff
@@ -129,9 +131,57 @@ struct TreeCmd {
     #[options(short = "f", help = "input Cargo.lock file to translate")]
     file: Option<PathBuf>,
 
-    /// Dependencies names to draw a tree for
-    #[options(free, help = "dependency names to draw trees for")]
-    dependencies: Vec<package::Name>,
+    /// Show exact package identities (checksums or specific source versions) when available
+    #[options(
+        short = "x",
+        help = "show exact package identies (checksums or specific source versions) when available"
+    )]
+    exact: bool,
+
+    // Show inverse dependencies rather than forward dependencies
+    #[options(
+        short = "i",
+        help = "show inverse dependencies _on_ a package, rather than forward dependencies _of_ a package"
```

Comment:
Could you also add `--invert` as an option, for consistency with `cargo tree -i` aka `cargo tree --invert`?"

Reply:
Oh, sure!